src/main/jhipster/src/main/webapp/app/metadata-manager/name-type/name-type.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { SERVER_API_URL } from '../../app.constants';
+import { ParamContext } from '../../shared/service/param.context';
+import { NameType, NameTypeDetails } from '../../shared/germplasm/model/name-type.model';
+import { createRequestOption } from '../../shared/model/request-util';
+
+@Injectable()
+export class NameTypeService {
+    constructor(private http: HttpClient,
+                private context: ParamContext) {
+
+    }
```

Comment:
Moving searchNameTypes to germplasm.service make sense, and probably we can replace that class in shared with this new one instead of having it here in metadata-manager folder. The services are only used in this module now, but maybe in the future they can be reused.
in other words: I propose moving `metadata-manager/name-type/name-type.service.ts` to `shared/name-type/service/name-type.service.ts `, replacing the old one

Reply:
Hi @nahuel-soldevilla I am agree. I will move it into shared/name-type/service.